tests/roles/ovn_adoption/tasks/main.yaml

```diff
@@ -73,6 +69,30 @@
     $CONTROLLER2_SSH sudo systemctl stop tripleo_ovn_cluster_northd.service
     $CONTROLLER3_SSH sudo systemctl stop tripleo_ovn_cluster_northd.service
 
+- name: set fact for oc run args
+  ansible.builtin.set_fact:
+    oc_run_args:
+      spec:
+        containers:
+        - image: "{{ ovn_image }}"
+          name: adoption-check
+          volumeMounts:
+          - mountPath: /backup
+            name: backup
+        template:
+          volumes:
+          - hostPath:
+              path: "{{ ovn_copy_tmp_dir }}"
+              type: Directory
+            name: backup
+
```

Comment:
so this might work for crc but its probably not ok for anything else
can we just use a PVC instead.

Reply:
bacialy we shoudl just create a volume with the same storage class we use for the control plan

yes that happens to ba a local volume with the default crc setup but
we should not actually force it tobe a hostPath volume

Reply:
that's a good idea, although is out of the patch scope, and belongs to future refactoring. But could be a single unit of work as soon as we're going to switch podman for oc run/exec globally (and start using the volumes, indeed)

Reply:
well honestly i would not merge this as is until this is fixed which is why i left the comment os i would like to see what other think